_pages/cfp.md

```diff
@@ -5,7 +5,8 @@ subtitle: Call for Proposals
 permalink: /cfp/
 ---
 
-Our CFP will open on August 14 2017 and close on September 15 2017! 
+Our call for proposals (CFP) for 15 minute lightning talks will open on August 14 2017 and close on September 15 2017!
```

Comment:
are we wanting people seeing this to start brainstorming ideas? Cause I think it's a lot easier to do that when you know the time frame